### PR TITLE
Add the Babel transform plugin

### PR DESCRIPTION
**Pull request by smfoote**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      path.node.value = '@ember/component';
+    }
+  }
+};
+
+module.exports = function() {
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        path.traverse(innerImportVisitor);
```
Comment (chadhietala) [inline index.js]: This is going to need to be a bit more sophisticated. Consider this case:

```js
import Component, { tracked } from '@glimmer/component';
```

Will become:

```js
import Component, { tracked } from '@ember/component';
```

This is not what we are going to want as we don't want the Component import to be changed.

Instead we want to emit.

```js
import Component from '@glimmer/component';
import { tracked } from '@ember/component';
```

We actually need to look which `ImportSpecifier`s are being imported and only emit the new declaration if `tracked` is imported from `@glimmer/component`.

File: tests/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const QUnit = require('qunit');
```
Comment (chadhietala) [inline tests/index.js:1]: I'm actually not sure if you need this since you are using the `qunit` binary to run the tests.

**Review by chadhietala — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (smfoote) [reply index.js]: This should be working now. Let me know if there are any other edge cases I should be addressing.

File: tests/index.js (hunk 2 above)
Comment (smfoote) [reply tests/index.js:1]: Seems to be necessary.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+module.exports = function() {
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        if (path.node.source.value === '@glimmer/component') {
+          let hasTrackedImport = false;
+          path.node.specifiers = path.node.specifiers.filter(specifier => {
+            if (specifier.type === 'ImportSpecifier' && specifier.imported.name === 'tracked') {
+              hasTrackedImport = true;
```
Comment (chadhietala) [inline index.js]: You're going to want to hoist the matched specifier and then use it below. Consider:

```js
import Component, { tracked as dataStalker } from '@glimmer/component';

export class extends Component {
  @dataStalker whatAWeirdName;
}
```

If you don't hoist the entire specifier you may run into issues where you've created shadowing.

Comment (smfoote) [reply index.js]: Good call
